scripts/code_coverage_report/generate_code_coverage_report/Package.resolved

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I'm not sure about committing the `Package.resolved` versus specifying the required versions in the Package.swift.